torch/csrc/api/include/torch/python/init.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <torch/csrc/utils/python_stub.h>
+
+namespace torch {
+namespace python {
+/// Initializes Python bindings for the C++ frontend.
+void init_bindings(PyObject* module);
```

Comment:
Is this really supposed to be part of the C++ API and not a function to be called internally from _C?

Reply:
You mean it should just be part of `csrc` and not in the `api` folder?

Reply:
I guess?